### PR TITLE
Control Label

### LINKED ISSUE #57: `Control Label` Component
Screen Shot 2022-08-22 at 8 02 14 PM

Make the active, hover, focus, and inactive states for the Control Label Component

### PR DESCRIPTION
**Pull request by ajo01**
Closes #57

### CODE REVIEW

File: src/components/input/checkbox/CheckboxComponent.js [hunk 1]
```diff
@@ -1,33 +1,51 @@
-import React, { useState, useEffect } from "react";
-import { Checkbox } from "@mui/material";
+import React, { useState, useEffect } from 'react';
+import { Checkbox, FormControlLabel } from '@mui/material';
+import styled from 'styled-components';
+import { colors } from '../../colors/Colors';
 
-import { colors } from "../../colors/Colors";
+const Label = styled.div``;
```
Comment (dchen150) [inline src/components/input/checkbox/CheckboxComponent.js:6]: is this necessary compared to just putting `div`?

File: src/components/input/checkbox/CheckboxComponent.js [hunk 2]
```diff
@@ -1,33 +1,51 @@
...
 
-const CheckboxComponent = ({ disabled, variant }) => {
+const CheckboxComponent = ({ disabled, variant, label }) => {
   const [checked, setChecked] = useState(false);
+  console.log('label', disabled);
```
Comment (dchen150) [inline src/components/input/checkbox/CheckboxComponent.js:10]: remove console.log

**Review by dchen150 — APPROVED** [review]
lgtm

File: src/components/input/checkbox/CheckboxComponent.js (hunk 2 above)
Comment (ajo01) [reply src/components/input/checkbox/CheckboxComponent.js]: Fixed

File: src/components/input/checkbox/CheckboxComponent.js (hunk 1 above)
Comment (ajo01) [reply src/components/input/checkbox/CheckboxComponent.js]: Nope, I forgot to switch it to a regular div. Fixed.

**Pull request merged**